_pages/address/12a-error-no-phone.html

```diff
@@ -0,0 +1,24 @@
+---
+layout: default
+title: Get started
+permalink: /error-no-phone/
+---
+
+<div class="container sm-pt3">
+  <div class="px2 py3 sm-py5 sm-px6 mx-auto sm-mb5 border-box card">
+    <form autocomplete="off" role="form" class="simple_form new_profile" id="new_profile" action="{{ "/address/" | prepend: site.baseurl }}" accept-charset="UTF-8">
+      <h1 class="h3 mt2">
+        We can’t verify your identity without a phone number you have access to.
+      </h1>
+      <p class="mb5">We’re working hard to add more ways to verify your identity.</p>
+      <div class="pb-tiny bg-blue col-2">
+      </div>
+      <div class="mb1 mt2 pt1">
+        <p>Visit <a href="">Railroad Retirement Board</a> for more information.</p>
```

Comment:
Should the RRB and view your account links go anywhere?

Reply:
oops, dammit, i should have commented on that—for the test, no we're letting those go since we're not hooking this up to anything. fyi there will be 2 more error pages coming that look almost identical, and will also have empty links.